core/embed/rust/src/zcash/allocator.rs

```diff
@@ -0,0 +1,25 @@
+/// Implementation of the global allocator. Enables usage of `core::alloc` crate.
```

Comment:
If we're gonna go with the GC allocator, I think it warrants a huge comment here documenting the invariants we need to maintain.